protocol/x/clob/keeper/orders_test.go

```diff
@@ -2084,16 +2077,6 @@ func TestInitStatefulOrders(t *testing.T) {
 
 			// Run the test and verify expectations.
 			ks.ClobKeeper.InitStatefulOrders(ks.Ctx)
-			for subaccountId, count := range expectedStatefulOrderCounts {
```

Comment:
keeping this check would have been nice since we could validate the count across more cases